mod_reforged/hooks/skills/actives/reload_bolt.nut

```diff
@@ -3,6 +3,7 @@
 	{
 		__original();
 		this.m.IsRemovedAfterBattle = false;
+		this.m.Order -= 100;	// Player expect this perk to come earlier than most other active skills because of muscle memory, so we move it forward a bit
```

Comment:
I'd just set this to `this.m.Order = ::Const.SkillOrder.OffensiveTargeted` as that is the order of shoot_bolt. This will ensure that this skill will **always** appear immediately after the shoot bolt skill which is the most intuitive.